### PR TITLE
Helm chart for Azure policy add-on for AKS Engine

### PR DESCRIPTION
**Pull request by RamyasreeChakka**
AKS Engine is a self managed Kubernetes cluster on Azure. The Azure Policy Add-on connects the Azure Policy service to the Gatekeeper policy admission controller on the AKS Engine. The policy add-on can be installed and enabled on the cluster using the helm chart in this PR.

### CODE REVIEW

File: policy-addon-kubernetes/helm-charts/azure-policy-addon-aks-engine/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Azure Policy Add-on for AKS Engine
```
Comment (pilor) [inline policy-addon-kubernetes/helm-charts/azure-policy-addon-aks-engine/README.md]: Can all of this go under a root "extensions" folder or similar to keep the repo organized?

Comment (pilor) [inline policy-addon-kubernetes/helm-charts/azure-policy-addon-aks-engine/README.md]: It seems weird to have this README 3 levels deep.  What is the "policy-addon-kubernetes" folder for?  Will there be other "helm-charts"?  

From someone who doesn't know what any of this is there isn't enough top-level documentation describing what the purpose of this folder tree is and how it should be used.

Comment (RamyasreeChakka) [reply policy-addon-kubernetes/helm-charts/azure-policy-addon-aks-engine/README.md]: Sure, will add a top level folder extensions and a readme file.

File: policy-addon-kubernetes/helm-charts/index.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    - compliance
+    - aks engine
+    - aks-engine
+    - kubernetes
+    maintainers:
+    - email: azuredg@microsoft.com
+      name: azuredg
+    name: azure-policy-addon-aks-engine
+    sources:
+    - https://github.com/Azure/azure-policy-kubernetes
```
Comment (EmandM) [inline policy-addon-kubernetes/helm-charts/index.yaml]: Do we want to link to this repo when it's private?

File: policy-addon-kubernetes/helm-charts/index.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    - azure-policy
+    - gatekeeper
+    - governance
+    - compliance
+    - aks engine
+    - aks-engine
+    - kubernetes
+    maintainers:
+    - email: azuredg@microsoft.com
+      name: azuredg
```
Comment (EmandM) [inline policy-addon-kubernetes/helm-charts/index.yaml]: What is this alias?

File: policy-addon-kubernetes/helm-charts/index.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    name: azure-policy-addon-aks-engine
+    sources:
+    - https://github.com/Azure/azure-policy-kubernetes
+    urls:
+    - https://raw.githubusercontent.com/Azure/azure-policy/master/policy-addon-kubernetes/helm-charts/azure-policy-addon-aks-engine-0.1.0.tgz
+    version: 0.1.0
+generated: "2019-10-22T10:39:56.7841011-07:00"
```
Comment (EmandM) [inline policy-addon-kubernetes/helm-charts/index.yaml]: Was this generated? Do we need this tag if this was hand built?

File: policy-addon-kubernetes/helm-charts/index.yaml (hunk 3 above)
Comment (RamyasreeChakka) [reply policy-addon-kubernetes/helm-charts/index.yaml]: This is Azure data plane policy team alias. For now using this alias, will update it when we have a new group alias.

File: policy-addon-kubernetes/helm-charts/index.yaml (hunk 4 above)
Comment (RamyasreeChakka) [reply policy-addon-kubernetes/helm-charts/index.yaml]: Generated. We don't need this tag.

File: policy-addon-kubernetes/helm-charts/index.yaml (hunk 2 above)
Comment (RamyasreeChakka) [reply policy-addon-kubernetes/helm-charts/index.yaml]: Sources is optional, I will remove it.

**Pull request merged**